localmemcache.go

```diff
@@ -127,9 +127,10 @@ func (mc *LocalMemcache) Get(key string) (*CacheItem, error) {
 		return nil, ErrCacheMiss
 	} else {
 		return &CacheItem{
-			Key:     key,
-			Value:   item.value,
-			casTime: item.addedAt,
+			Key:        key,
+			Value:      item.value,
+			casTime:    item.addedAt,
+			Expiration: item.expires.Sub(item.addedAt),
```

Comment:
I'm iffy on this line and would like feedback. I'm returning the originally set expiration duration, but I could see an argument for returning a (potentially negative) duration indicating when this expired. I added this to test that subsequent increments do not modify the TTL